sklearn/covariance/tests/test_covariance.py

```diff
@@ -196,6 +192,18 @@ def test_ledoit_wolf():
     assert(lw.precision_ is None)
 
 
+def test_ledoit_wolf_large():
```

Comment:
Your test is not a regression test. Before, for >1000 features the algorithm just refused to handle the data by default. This was supposed to be a regression test for this. Sorry, I should have documented better.